ApiDoctor.Validation/ExtensionMethods.cs

```diff
@@ -573,7 +573,43 @@ public static bool IsHeaderBlock(this Block block)
                     return false;
             }
         }
+        /// <summary>
+        /// Uses the Damerau-Levenshtein distance algorithm which calculates how different one string is from another 
+        /// </summary>
+        /// <param name="s"></param>
+        /// <param name="t"></param>
+        /// <returns></returns>
+        public static int StringDistance(this string s, string t)
```

Comment:
any reason why we're rolling our own implementation here as opposed to bringing on a dependency?
https://www.nuget.org/packages/Fastenshtein
https://www.nuget.org/packages/FuzzySharp
(and probably many more)

Reply:
Good point. Will use Fastenshtein.